chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author Chaos --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
@PlasmaPower yeah it will look a bit stange, but it's just my avatar 2 times: [example PR](https://github.com/chaosbot/Chaos/pull/326)
if you'll run `git log --author Chaos` my commit will be there